kmip/demos/pie/create_key_pair.py

```diff
@@ -23,6 +22,8 @@
 
 
 if __name__ == '__main__':
+    logger = utils.build_console_logger(logging.INFO)
```

Comment:
How does this work for the user? Can they set the log level and logging file?

If not then the logger.debug statement will never be seen, and could be removed.